File: src/main/java/net/gtaun/shoebill/event/vehicle/VehicleCreateEvent.java
package net.gtaun.shoebill.event.vehicle;

import net.gtaun.shoebill.object.Vehicle;

/**
 * Created by marvin on 01.11.14.
 */
public class VehicleCreateEvent extends VehicleEvent {
    public VehicleCreateEvent(Vehicle vehicle) {
        super(vehicle);
    }
}

File: src/main/java/net/gtaun/shoebill/event/vehicle/VehicleCreateEvent.kt
package net.gtaun.shoebill.event.vehicle

import net.gtaun.shoebill.entities.Vehicle

/**
 * This event will be called when a new [Vehicle] has been created.
 *
 * @author Marvin Haschker
 */
class VehicleCreateEvent(vehicle: Vehicle) : VehicleEvent(vehicle)
